src/Microsoft.AspNetCore.Mvc.Core/Internal/ControllerActionInvokerCache.cs

```diff
@@ -45,24 +45,28 @@ private InnerCache CurrentCache
             }
         }
 
-        public IFilterMetadata[] GetFilters(ActionContext actionContext)
+        public Entry GetActionInvokerCachedItem(ControllerContext controllerContext)
         {
             var cache = CurrentCache;
-            var actionDescriptor = actionContext.ActionDescriptor;
+            var actionDescriptor = controllerContext.ActionDescriptor;
 
-            CacheEntry entry;
+            Entry entry;
             if (cache.Entries.TryGetValue(actionDescriptor, out entry))
             {
-                return GetFiltersFromEntry(entry, actionContext);
+                return entry;
             }
 
+            // create action method executor
```

Comment:
Don't add comments like these. This doesn't explain anything that isn't obvious from the code